Formula/bsdrpcgen.rb

```diff
@@ -0,0 +1,20 @@
+class Bsdrpcgen < Formula
```

Comment:
As for the name, I'm not sure if just rpcgen would be better because there's a different rpcgen from glibc. At the first glance they might have common roots, but explicit naming could be less confusing probably.